google-cloud-clients/google-cloud-bigtable/pom.xml

```diff
@@ -18,10 +18,23 @@
     <site.installationModule>google-cloud-bigtable</site.installationModule>
   </properties>
   <dependencies>
+    <!-- Production dependencies -->
```

Comment:
Please mention that these are alphabetical, so that future devs know what pattern to follow.

Reply:
Done

Reply:
As long as alphabetical works, that's fine. This does affect which versions of transitive dependencies maven selects though. That is, the ordering is significant and changes in order can change the classpath and fix or break the build.